index.js

```diff
@@ -298,7 +302,10 @@ const run = async userOptions => {
       } else if (options.asyncJs) {
         await asyncJs({ page });
       }
-      const filePath = path.join(destinationDir, route);
+      const routePath = options.publicPath !== '/'
+        ? route.replace(options.publicPath, '')
```

Comment:
I feel like there is possible edge case when public path ends with `/` or not

Reply:
yea, trailing slash should be removed from public path to make this safer. i will make that change.